app/qml/map/RecordingTools.qml

```diff
@@ -49,10 +49,6 @@ Item {
     positionKit: __positionKit
     activeLayer: __activeLayer.vectorLayer
     activeFeature: root.activeFeature
-    variablesManager: __variablesManager
-
-    // Bind variables manager to know if we are centered to GPS or not when evaluating position variables
-    onIsUsingPositionChanged: __variablesManager.useGpsPoint = isUsingPosition
```

Comment:
This line should most likely not be removed. It allows `variablesManager` to fill the `from_gps` variable.